robottelo/ui/computeresource.py

```diff
@@ -1,72 +1,113 @@
-                if self.wait_until_element(locators["resource.url"]):
-                    self.find_element(locators["resource.url"]).send_keys(url)
-        if libvirt_display is not None:
-            display = self.find_element(locators["resource.libvirt_display"])
-            Select(display).select_by_visible_text(libvirt_display)
-        if libvirt_set_passwd is False:
-            self.find_element(
-                locators["resource.libvirt_console_passwd"]).click()
-        self.find_element(locators["resource.test_connection"]).click()
-        self.wait_for_ajax()
+            Select(
+                self.find_element(locators["resource.provider_type"])
+            ).select_by_visible_text(provider_type)
+        for parameter_name, parameter_value, parameter_type in parameter_list:
+            param_locator = '.'.join((
+                'resource',
+                (parameter_name.lower()).replace(' ', '_')
+            ))
+            self.wait_until_element(locators[param_locator])
+            if parameter_type == 'field':
+                self.find_element(
+                    locators[param_locator]).send_keys(parameter_value)
+            elif parameter_type == 'select':
+                Select(
+                    self.find_element(locators[param_locator])
+                ).select_by_visible_text(parameter_value)
+            elif parameter_type == 'checkbox':
+                if (self.find_element(locators[param_locator]).is_selected() !=
+                        parameter_value):
+                    self.click(locators[param_locator])
+            elif parameter_type == 'special select':
+                button_locator = '.'.join((
+                    'resource',
+                    (parameter_name.lower()).replace(' ', '_'),
+                    'button'
+                ))
+                self.click(locators[button_locator])
+                self.find_element(
+                    locators[param_locator]
+                ).select_by_visible_text(parameter_value)
+
+    def _configure_orgs(self, orgs, org_select):
+        """
+        Provides configuration capabilities for compute resource organization.
+        Next format should be used: orgs=['Aoes6V', 'JIFNPC'], org_select=True
 
-    def create(self, name, orgs, description=None, org_select=True,
-               provider_type=None, url=None, user=None,
-               password=None, region=None, libvirt_display=None,
-               libvirt_set_passwd=True, tenant=None):
+        """
+        if orgs:
+            self.configure_entity(
+                orgs,
+                FILTER['cr_org'],
+                tab_locator=tab_locators["tab_org"],
+                entity_select=org_select
+            )
+
+    def _configure_locations(self, locations, loc_select):
+        """
+        Provides configuration capabilities for compute resource location
+        Next format should be used: locations=['Default Location'],
+        loc_select=True
```

Comment:
I'm having trouble understanding this docstring. Do you mean something like this?

```
        """Provides configuration capabilities for compute resource location

        The following format should be used::

            locations=['Default Location'], loc_select=True

        """
```